app/src/main/java/de/danoeh/antennapod/fragment/NavDrawerFragment.java

```diff
@@ -366,7 +370,10 @@ private void loadData() {
     @Override
     public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
         int viewType = parent.getAdapter().getItemViewType(position);
-        if (viewType != NavListAdapter.VIEW_TYPE_SECTION_DIVIDER) {
+        if (viewType == NavListAdapter.VIEW_TYPE_SECTION_DIVIDER) {
```

Comment:
Please check if it is actually filtered (just to be sure)

Reply:
Changed it to an else statement